lmdeploy/pytorch/models/module_map.py

```diff
@@ -149,6 +149,12 @@
     f'{LMDEPLOY_PYTORCH_MODEL_PATH}.internvl.InternVLChatModel'
 })
 
+# mono-internvl
+MODULE_MAP.update({
+    'InternLM2VEForCausalLM':
+    f'{LMDEPLOY_PYTORCH_MODEL_PATH}.internlm2_ve.InternLM2VEForCausalLM',
```

Comment:
may update this supported list as well
https://github.com/InternLM/lmdeploy/blob/2bed0182a8bac22e815ddfb6e00be1f680dea927/lmdeploy/pytorch/supported_models.py#L64